tests/unit/mpa/modules/models/scalar_schedulers/test_schedulers.py

```diff
@@ -0,0 +1,126 @@
+"""Test schedulers."""
+
+
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+
+from otx.mpa.modules.models.scalar_schedulers import (
+    ConstantScalarScheduler,
+    PolyScalarScheduler,
+    StepScalarScheduler,
+)
+
+
+class TestSchedulers:
+    """Test schedulers."""
+
+    def test_constant_scalar_scheduler(self):
+        """Test constant scalar scheduler."""
+        scheduler = ConstantScalarScheduler(scale=30.0)
+
+        # learning rate should not change over time
```

Comment:
Can you add this to either docstring or name the test function using this expression?